### PR TITLE
fix: lyric example running again with adaptions in core

### PR DESCRIPTION
**Pull request by maximilianwerk**
This makes the lyric search run again. Be aware, that https://github.com/jina-ai/jina/pull/933 is crucial for making it run again.

### CODE REVIEW

File: multires-lyrics-search/app.py [hunk 1]
```diff
@@ -6,16 +6,17 @@
...
 from jina.proto import jina_pb2
 
-num_docs = os.environ.get('MAX_DOCS', 50000)
+num_docs = os.environ.get('MAX_DOCS', 500)
 
 
 def config():
-    parallel = 6 if sys.argv[1] == 'index' else 1
-    shards = 8
+    parallel = 1 if sys.argv[1] == 'index' else 1
```
[the remark this answers is not in the document]
Comment (maximilianwerk) [reply multires-lyrics-search/app.py:18]: not sure about it. having a parallelism > 1 requires a machine with at least 9 cores to actually get a speed-up. The `TransformerTorchEncoder` with `distilbert-base-cased` utilizes 4 cores and some power is needed for the rest of the machine. Thus, with 8 cores and a `parallel=2`, we already have a lot of low-level-cache data loading, which makes the overall performance worse.

File: multires-lyrics-search/app.py [hunk 2]
```diff
@@ -25,9 +26,9 @@ def config():
 
 
 def input_fn():
-    with open('data/lyrics-data.csv', newline='', encoding='utf-8') as f:
+    with open('toy-data/lyrics-toy-data1000.csv', newline='', encoding='utf-8') as f:
```
[the remark this answers is not in the document]
Comment (maximilianwerk) [reply multires-lyrics-search/app.py:29]: :+1:

File: multires-lyrics-search/app.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoanFM) [reply multires-lyrics-search/app.py:18]: I was just thinking about mantaining it as it was, just a reminder ... if u think is better like this I am okey

File: multires-lyrics-search/pods/encode.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 !TransformerTorchEncoder
 with:
-  pooling_strategy: cls
+  pooling_strategy: auto
```
[the remark this answers is not in the document]
Comment (maximilianwerk) [reply multires-lyrics-search/pods/encode.yml:3]: which typo do you talk about? `pooling_strategy` is right. And `cls` is also the default parameter in the `pooling_strategy`.

[the remark this answers is not in the document]
Comment (JoanFM) [reply multires-lyrics-search/pods/encode.yml:3]: yes, good!

File: multires-lyrics-search/app.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nan-wang) [reply multires-lyrics-search/app.py:18]: Let's set `parallel=2`. Or else, this line of codes looks redundant.

**Comment by nan-wang** [comment]
related to https://github.com/jina-ai/jina/pull/933

**Pull request merged**